src/VisualStudio/Core/Def/Microsoft.VisualStudio.LanguageServices.csproj

```diff
@@ -134,25 +126,14 @@
     <None Include="ManagedEditAndContinueService.vsdconfigxml" />
   </ItemGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.CSharp" />
-    <Reference Include="PresentationCore" />
-    <Reference Include="PresentationFramework" />
-    <Reference Include="System" />
-    <Reference Include="System.Core" />
     <Reference Include="System.Design" />
-    <Reference Include="System.Drawing" />
-    <Reference Include="System.Runtime.Serialization" />
     <Reference Include="System.Windows.Forms" />
-    <Reference Include="System.Xaml" />
-    <Reference Include="System.Xml" />
-    <Reference Include="System.Xml.Linq" />
     <Reference Include="UIAutomationProvider" />
     <Reference Include="UIAutomationTypes" />
-    <Reference Include="WindowsBase" />
-    <Reference Include="WindowsFormsIntegration" />
     <PackageReference Include="EnvDTE" Version="$(EnvDTEVersion)" />
     <PackageReference Include="EnvDTE80" Version="$(EnvDTE80Version)" />
     <PackageReference Include="Microsoft.CodeAnalysis.Elfie" Version="$(MicrosoftCodeAnalysisElfieVersion)" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.CSharp" Version="$(MicrosoftCSharpVersion)" />
```

Comment:
Why is Microsoft.CSharp switching from Reference to PackageReference?

Reply:
I am thinking a PackageReference is preferred over Reference. Ideally we would only have PackageReferences.

Reply:
I'm not sure that's a good idea in general for .NET Framework. These will become dependencies on pack and create bigger package graphs that can slow things down and complicate things unnecessarily. I would stick to the .NET Framework References in .NET Framework projects.

Reply:
ok

Reply:
I would also expect PackageReference. If we use Reference, it means we are manually applying part of the NuGet algorithm just to work around the SDK being slow. If the SDK is slow, we should address that problem directly.

> will become dependencies on pack

This _is_ a dependency, but if we still want to suppress it for some reason, we have a way to do that.